app/lib/dags/non_host_alignment.json.erb

```diff
@@ -50,6 +50,9 @@
         "chunk_size": 15000,
         "max_concurrent": 3,
         "environment": "prod"
+        <% if @attribute_dict[:index_dir_suffix] %>
```

Comment:
maybe index_dir_date or index_dir_tag would be a better name if we're using it for paths like `s3://idseq-database/alignment_data/2018-04-01-utc-1522569777-unixtime__2018-04-04-utc-1522862260-unixtime/nt` ?

Reply:
use suffix to leave the options open in case we include not only date in the future.